etcdserver/cluster_util.go

```diff
@@ -65,34 +64,34 @@ func getClusterFromRemotePeers(urls []string, logerr bool, tr *http.Transport) (
 		resp, err := cc.Get(u + "/members")
 		if err != nil {
 			if logerr {
-				log.Printf("etcdserver: could not get cluster response from %s: %v", u, err)
+				plog.Warningf("could not get cluster response from %s: %v", u, err)
```

Comment:
if a member is offline, then there will be a warning. and it will not affect the future operations.